CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Version changelog
 
+## 1.36.0
+
+### New Features and Improvements
+* Added `databricks_volumes` as data source  ([#3150](https://github.com/databricks/terraform-provider-databricks/pull/3150)).
+* Introduced Generic Settings Resource ([#2997](https://github.com/databricks/terraform-provider-databricks/pull/2997)).
+* Fixed updating owners for UC resources ([#3189](https://github.com/databricks/terraform-provider-databricks/pull/3189)).
+* Validated metastore id for databricks_grant and databricks_grants resources ([#3159](https://github.com/databricks/terraform-provider-databricks/pull/3159)).
+* Updated ([#3192](https://github.com/databricks/terraform-provider-databricks/pull/3192)).
```

Comment:
This should be in the "Internal changes" section and also we need to have a full description